code/game/machinery/computer/robot.dm

```diff
@@ -3,7 +3,7 @@
 	desc = "Used to remotely lockdown or detonate linked Cyborgs and Drones."
 	icon_screen = "robot"
 	icon_keyboard = "rd_key"
-	req_access = list(ACCESS_ROBOTICS)
+	req_access = list(ACCESS_AI_UPLOAD)
```

Comment:
fuuuuuck no

Reply:
the rest is fine but this is painful as shit when it comes to bad cyborgs; going to result in a lot more "we killed the borg, re-assemble when it stops being a shitter" when bad cyborg players occur

Reply:
at the very least, make this a new access that only the Roboticists and the RD get?

Reply:
> at the very least, make this a new access that only the Roboticists and the RD get?

so robotics access.

Reply:
> the rest is fine but this is painful as shit when it comes to bad cyborgs; going to result in a lot more "we killed the borg, re-assemble when it stops being a shitter" when bad cyborg players occur

you have tipping, so killing is excessive in this situation

Reply:
@Kubisopplay the more i think about it the less i like restricting the access to upload, but i already told you removing detonate was the only thing i require right now. so fix tests and we can get it tm'd and see if it becomes a problem